pkg/resource/deploy/deployment.go

```diff
@@ -56,10 +56,8 @@ type Options struct {
 	Parallel                  int        // the degree of parallelism for resource operations (<=1 for serial).
 	Refresh                   bool       // whether or not to refresh before executing the deployment.
 	RefreshOnly               bool       // whether or not to exit after refreshing.
-	RefreshTargets            UrnTargets // The specific resources to refresh during a refresh op.
-	ReplaceTargets            UrnTargets // Specific resources to replace.
-	DestroyTargets            UrnTargets // Specific resources to destroy.
-	UpdateTargets             UrnTargets // Specific resources to update.
+	Targets                   UrnTargets // If specified, only update specified resources.
+	ReplaceTargets            UrnTargets // If specified, replace these resources.
```

Comment:
Should we comment here that if targets is set then ReplaceTargets must be a subset?